msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -43,7 +43,9 @@ At a high level, the import process consists of the following:
 
```

Comment:
3 and 4th line should be swapped in the following block.  This comment contains the correct sequence.

### Analyze and prepare message data
✔ Review the third-party data to decide what will be migrated. 
✔ Extract the selected data from the third-party chat system. 
✔ Map the third-party chat structure to the Teams structure.
✔ Convert import data into format needed for migration.